docs/source/en/using-diffusers/other-formats.mdx

```diff
@@ -123,4 +123,71 @@ pipeline.to("cuda")
 placeholder_token = "<my-funny-cat-token>"
 prompt = f"two {placeholder_token} getting married, photorealistic, high quality"
 image = pipeline(prompt, num_inference_steps=50).images[0]
-```
\ No newline at end of file
+```
+
+## A1111 LoRA files
+
+[Automatic1111](https://github.com/AUTOMATIC1111/stable-diffusion-webui) (A1111) is a popular web UI for Stable Diffusion that supports model sharing platforms like [Civitai](https://civitai.com/). Models trained with the Low-Rank Adaptation (LoRA) technique are especially popular because they're fast to train and have a much smaller file size than a fully finetuned model. However, A111 LoRA files are formatted differently from the LoRA file format used by 🤗 Diffusers, which means you'll need to use the [`~LoraLoaderMixin.load_lora_weights`] to load A111 LoRA files.
```

Comment:
nits
```suggestion
[Automatic1111](https://github.com/AUTOMATIC1111/stable-diffusion-webui) (A1111) is a popular web UI for Stable Diffusion that supports model sharing platforms like [Civitai](https://civitai.com/). Models trained with the Low-Rank Adaptation (LoRA) technique are especially popular because they're fast to train and have a much smaller file size than a fully finetuned model. However, A1111 LoRA files are formatted differently from the LoRA file format used by 🤗 Diffusers, which means you'll need to use the [`~LoraLoaderMixin.load_lora_weights`] to load A1111 LoRA files.
```

Reply:
>However, A111 LoRA files are formatted differently from the LoRA file format used by 🤗 Diffusers, which means you'll need to use the [`~LoraLoaderMixin.load_lora_weights`] to load A111 LoRA files.

For this part, `LoraLoaderMixin.load_lora_weights` can be used with the Diffusers' LoRA format as well, so how about explaining it a bit more logically like this?

```suggestion
[Automatic1111](https://github.com/AUTOMATIC1111/stable-diffusion-webui) (A1111) is a popular web UI for Stable Diffusion that supports model sharing platforms like [Civitai](https://civitai.com/). Models trained with the Low-Rank Adaptation (LoRA) technique are especially popular because they're fast to train and have a much smaller file size than a fully finetuned model. 🤗 Diffusers now supports the loading of A1111 LoRA, so you can load it using [~LoraLoaderMixin.load_lora_weights].
```